teamcity/datadog_checks/teamcity/teamcity.py

```diff
@@ -16,15 +15,22 @@
 
 
 class TeamCityCheck(AgentCheck):
-    HEADERS = {'Accept': 'application/json'}
+    HEADERS = {"Accept": "application/json"}
     DEFAULT_TIMEOUT = 10
-    NEW_BUILD_URL = "http://{server}/guestAuth/app/rest/builds/" +\
-                    "?locator=buildType:{build_conf},sinceBuild:id:{since_build},status:SUCCESS"
-    LAST_BUILD_URL = "http://{server}/guestAuth/app/rest/builds/" +\
-                     "?locator=buildType:{build_conf},count:1"
-    NEW_BUILD_URL_AUTHENTICATED = "http://{server}/httpAuth/app/rest/builds/" +\
-                                  "?locator=buildType:{build_conf},sinceBuild:id:{since_build},status:SUCCESS"
-    LAST_BUILD_URL_AUTHENTICATED = "http://{server}/httpAuth/app/rest/builds/?locator=buildType:{build_conf},count:1"
+
+    # E
```

Comment:
🤔

Reply:
😆 That's what I get for not manually saving.